platform/brains2/src/lib/leds.cpp

```diff
@@ -147,11 +137,10 @@ static uint32_t show_pixels(const Pixel *const pixels, const int pixel_count) {
   return true;
 }
 
-void setBrightness(int brightness) {
-  _brightness = brightness;
-}
+void setBrightness(int brightness) { _brightness = brightness; }
```

Comment:
this is a matter of style. We don't do single line statements so I suggest leaving the old version intact.

Reply:
It's a result of me using the `clang-format` with default configuration to auto-format the code.
For me, either style is fine, but whatever style we use should be enforced by an auto-formatter in my opinion. For C and C++, `clang-format` is by far the best I've come across, and it's also quite configurable.
We can even generate a configuration file based on then google style guide with `clang-format -style=llvm -dump-config > .clang-format`.
I fixed this particular case in this branch (also because we shouldn't have unrelated changes), and made this issue: https://github.com/datomusic/duo-imxrt/issues/121